std_msgs/Header header
geometry_msgs/Point[] park_points
std_msgs/Bool can_park